src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -42,9 +42,21 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
-                ->scalarNode('title')->defaultValue('')->info('The title of the API.')->end()
-                ->scalarNode('description')->defaultValue('')->info('The description of the API.')->end()
-                ->scalarNode('version')->defaultValue('0.0.0')->info('The version of the API.')->end()
+                ->scalarNode('title')
+                    ->info('The title of the API.')
+                    ->cannotBeEmpty()
+                    ->defaultValue('')
+                ->end()
+                ->scalarNode('description')
+                    ->info('The description of the API.')
+                    ->cannotBeEmpty()
+                    ->defaultValue('')
+                ->end()
+                ->scalarNode('vesion')
```

Comment:
`version`

Reply:
Oups. Goog catch. Thanks (We should never edit code on Github directly :p)